pkg/controller/backup_controller.go

```diff
@@ -120,6 +122,7 @@ func NewBackupController(
 		volumeSnapshotLister:        volumeSnapshotLister,
 		volumeSnapshotContentLister: volumeSnapshotContentLister,
 		newBackupStore:              persistence.NewObjectBackupStore,
+		defaultVolumesToRestic:      defaultVolumesToRestic,
```

Comment:
Nit: it'd be good to place this consistently under the `defaultBackupLocation`, as with the other two references.